docs/Users_Guide/release-notes.rst

```diff
@@ -12,20 +12,22 @@ METdataio Release Notes
 METdataio Version 2.1.0-beta1 release notes (20230228)
 ------------------------------------------------------
 
+  .. dropdown:: New Functionality
+     
+     None 
 
-* New Functionality:
+  .. dropdown:: Enhancements
+     
+     * **Upgrade to using Python 3.10.4** (`#176 <https://github.com/dtcenter/METdataio/issues/176>`_)
 
-* Enhancements:
-    * **Upgrade to using Python 3.10.4**
-      (`#176 <https://github.com/dtcenter/METdataio/issues/176>`_)
+  .. dropdown:: Internal
+     
+     * **Add 'LICENSE.md' to the METdatio repo** (`#184 <https://github.com/dtcenter/METdataio/issues/184>`_)
```

Comment:
```suggestion
     * Add 'LICENSE.md' to the METdatio repo (`#184 <https://github.com/dtcenter/METdataio/issues/184>`_)
```

Reply:
Please remove the bolding from "Add ‘LICENSE.md’ to the METdatio repo ([#184](https://github.com/dtcenter/METdataio/issues/184))" to be consistent with MET and because it is not a major/important change that the user's attention needs to be drawn to (that is why we bold certain items).

Reply:
I've removed the bolding.  (I just noticed the commit suggestion button.  I'll use it next time.)